WordPress/Classes/Networking/ServiceRemoteREST.h

```diff
@@ -4,6 +4,7 @@ typedef enum
 {
     ServiceRemoteRESTApiVersion_1_1 = 1001,
     ServiceRemoteRESTApiVersion_1_2 = 1002,
+    ServiceRemoteRESTApiVersion_1_3 = 1003,
```

Comment:
Oops. We ended up using v1 of the REST API for the endpoint, not v3.  Old code. Will purge.